config/database.yml

```diff
@@ -1,25 +1,20 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV["DB_POOL"] || ENV.fetch('PUMA_MAX_THREADS', 5).to_i %>
+
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: <%= ENV['DATABASE_NAME'] || 'bookstore-api' %>
+  password: <%= ENV['DATABASE_PASSWORD'] || '' %>
+  host: <%= ENV['DATABASE_HOST'] || '127.0.0.1' %>
+  username: <%= ENV['DATABASE_USERNAME'] || '' %>
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: bookstore-api_test
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  url: <%= ENV['DATABASE_URL'].present? ? "#{ENV['DATABASE_URL']}?prepared_statements=false" : '' %>
```

Comment:
Why did you choose to disable prepared statements in production?

Reply:
I will be honest here. I just copied `database.yml` from one of the project and changed it as I needed and didn't noticed `prepared_statements` parameter here. :see_no_evil:
We were facing issue of too many prepared statements getting cached that project, as we use lot of `IN` queries there. That is why choose to disable prepared statements.

I came to know about this from https://devcenter.heroku.com/articles/postgres-logs-errors#out-of-memory-errors article.